en/components/general/wpf_to_angular_guide/one_way_binding.md

```diff
@@ -0,0 +1,115 @@
+```html
+<h2 innerHTML="{{ text }}"></h2>
+```
+There two important things about `interpolation`. First, everything inside curly braces is rendered as a `string`. 
+And second, everything inside the curly braces is referred to `template expression`. This allows to do more complex things, such as a concatenation. For example, let's concatenate some text with the value of the `text` property:
+```html
+<h2>{{"Welcome to " + text }}</h2>
+```
+Using the template expression also allows to data bind to java script properties and methods. For example, we can data bind to the `text` propetry length which will result - 20:
+```html
+<h2>{{ text.length }}</h2>
+```
+We can also data bind to methods of that property, for example to `toUpperCase()`:
+```html
+<h2>{{ text.toUpperCase() }}</h2>
+```
+This is much more powerful than data binding in WPF, and a lot easier to use. We can even make mathematical calculation inside the template expression. For example, we can simply put 2+2 into the expression, and it will display the result - 4:
+```html
+<h2>{{ 2 + 2 }}</h2>
+```
+Another possible thing, is to data bind to actual methods from the typescript file. Here is a short example how to achieve this:
+```html
+<h2>{{ getTitle() }}</h2>
+```
+This `getTitle()` is a method defined in the typescript file. The result on the page from above binding, is the returned value of that method:
+```typescript
+getTitle() {
+  return 'Simple Title';
+}
+```
+
+Although the `interpolation` may look powerful, it has some limitations - it only represents a string.
+So let's create a simple boolean property in the component class:
+```typescript
+export class SampleComponent implements OnInit {
+
+  text = 'IgniteUI for Angular';
+  isDisabled = false;
+  constructor() { }
+...
+```
+Then create a simple `input` of type text, and bind the `isDisabled` to its disabled property:
+```html
+<input type="text" disabled="{{ isDisabled }}">
+```
+The expected result is, that the `input` state should be enabled, but it's disabled. This is because the `interpolation` returns a string, but the input's disabled property is boolean and it requires a boolean value. 
+Here it comes the `property binding` to make things work. It has a little different syntax:
+
+### Property Binding
+
+```html
+<input type="text" [disabled]="isDisabled">
+```
+The actual property name is wrapped into square brackets, and its value does not contain curly braces - just the name of the property that is bind to. Now, the input's disabled property is data bind to a boolean result, not a string. The isDisabled value is false, and running the app would display the input into enabled state. So, it's important to keep in mind that, whenever binding to a property relies on the data type result, then a `property binding` should be used. And if the binding simply relies on a string, then `interpolation` should be used.
```

Comment:
1. ... just the name of the property to bind to ...
2. the input's `disabled` property is ~~data bind~~ bound to a boolean result, not to a string.
3. The `isDisabled` value - mark the properties

Reply:
Addressed